src/logo-system/logo-elements/index.md

```diff
@@ -50,14 +50,14 @@ The size of the crown can be adjusted depending on context. For example, when be
 Scaling should follow this rule:
 Wordmark dot = 2× crown dot
 
-![TODO](./propotional-scaling-desktop.svg)
+![Dots from the crown are used to show the correct scale between the crown and wordmark.](./propotional-scaling-desktop.svg)
```

Comment:
(Optional) I first thought this was incorrect. I had interpreted to show the correct size of the dot between the "GOV" and "UK". But when I thought about it, I think what you wrote means the same thing just from a different angle.